desktop/release-notes.md

```diff
@@ -24,13 +24,94 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.0
+
+{% include release-date.html date="2022-01-12" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Extensions have moved from Beta to GA
+- Quick Search has moved from experimental to GA
+- Extensions are now included in Quick Search
+- Analyzing large images is now up to 4x faster
+
+### Upgrades
+
+- [Compose v2.15.1](https://github.com/docker/compose/releases/tag/v2.15.1)
+- [Containerd v1.6.14](https://github.com/containerd/containerd/releases/tag/v1.6.14)
+- [Docker Engine v20.10.22](https://docs.docker.com/engine/release-notes/#201022)
+- [Docker Scan v0.23.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.23.0)
+- [Go 1.19.4](https://github.com/golang/go/releases/tag/go1.19.4)
+
+### Bug fixes and enhancements
+
```

Comment:
Forgot to add notes to https://github.com/docker/pinata/pull/19937 - maybe something like `AF_VSOCK syscalls from within containers get blocked` in a separate security section. 

It's actually related to a vuln report (https://docker.atlassian.net/browse/POS-1175), basically same `AF_VSOCK` issue with accessing the host filesystem without `-v` that we had before but for WSL2 (where we can't do the one-connection hack)

Reply:
No related CVE?

Reply:
No CVE yet as there’s still the problem with socketcall on 32-bit as seccomp doesn’t dereference pointer args. Discussing on #desktop-security-private how to go about it